content/_announcements/jan2020-new-website.md

```diff
@@ -4,25 +4,37 @@ title: Our new website
 
-## Heading
+Lundeby has built a site with this unusual toolkit before. In 2018, after evaluating "a number of tools," including static site generators and CCMS systems, she decided to use Jekyll to build a site for the documentation she maintains for Veeva Systems, a Pleasanton-based cloud-computing company that builds applications for the pharmaceutical and life sciences industry.
 
-Portland hexagon put a bird on it green juice. Yr marfa twee man bun. Hexagon YOLO stumptown, succulents yr chia man bun. Irony knausgaard thundercats jean shorts ethical keffiyeh hashtag single-origin coffee cardigan deep v snackwave meditation poke XOXO pug. Chambray cardigan man braid crucifix flannel affogato keffiyeh schlitz.
+She chose Jekyll because it had a largest number of users and the largest support presence of all the likely options. That doesn't mean the job was easy.
 
-Tumeric leggings mlkshk tote bag. Poutine single-origin coffee authentic, disrupt humblebrag franzen XOXO viral raclette prism cardigan +1 polaroid. Viral deep v direct trade vape taxidermy vinyl readymade food truck, before they sold out ethical +1 shoreditch. Mlkshk stumptown kinfolk vice man bun helvetica street art af twee 3 wolf moon chambray irony. Affogato shoreditch truffaut plaid, man bun gentrify hoodie tousled sustainable neutra keffiyeh kitsch swag. Mixtape polaroid woke art party austin jianbing. Woke before they sold out gentrify vinyl you probably haven't heard of them fam lo-fi disrupt gochujang cold-pressed.
+With just a little previous exposure to HTML and CSS, Lundeby waded into Jekyll, only to start over again four or five times, after realizing that each subsequent theme she chose was not going to do the job the way she wanted. She eventually decided to build her own custom theme, which is what her Veeva docs site uses today.
 
-Oh. You need a little dummy text for your mockup? How quaint.
+Lundeby decided to use Bootstrap because the framework provides components like navbars and tables of contents without the need to write all the code for them. It also powers the reusable venue cards on our events pages, and the embedded Google maps.
+"I did not think I had time to build out the JavaScript and CSS for a bunch of UI components," she said.
 
-I bet you’re still using Bootstrap too…
\ No newline at end of file
+Lundeby's second project with Jekyll and Bootstrap went much easier than the first. She made the first commit December 19, shortly after the urgency of building a site became apparent. Less than a month later this site was visible on the web, much the way it looks today.
+
+"I benefitted from having built a Jekyll site before," she said. "There was a lot more swearing that time."
+
+Write The Docs Bay Area will still use Meetup.com as long as it's available for free, but now members can also use this site to learn about upcoming events and to RSVP to them.
```

Comment:
So I don't think this is actually true...I thought that Write The Docs as the larger organization covered the fees associated with Meetup.com for the local meetup groups.

Reply:
I'll change it to: 
"As long as it's practical."

People do find us by searching there for documentation, so we can post there if it doesn't cost us to do so, but we can also use our own site so that members will never have to pay to RSVP.